# Add extra channel info
#
# Copyright (C) 2025 Kazutoiris
# This file is licensed under the Anti 996 License.
# You may not use this file except in compliance with the License.
# It can be found in the LICENSE file.

--- a/components/version_ui_strings.grdp
+++ b/components/version_ui_strings.grdp
@@ -4,10 +4,10 @@
     About Version
   </message>
   <message name="IDS_VERSION_UI_OFFICIAL" desc="official build on the about:version page">
-    Official Build, ungoogled-chromium
+    Official Build, ungoogled-chromium, GitHub@kazutoiris Mod
   </message>
   <message name="IDS_VERSION_UI_UNOFFICIAL" desc="unofficial build on the about:version page">
-    Developer Build, ungoogled-chromium
+    Developer Build, ungoogled-chromium, GitHub@kazutoiris Mod
   </message>
   <message name="IDS_VERSION_UI_32BIT" desc="32-bit on the chrome://version page">
     (32-bit)
